src/test/kotlin/org/rust/ide/annotator/fixes/AddSelfTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.rust.ide.annotator.fixes
+
+import org.junit.Test
+import org.rust.ide.annotator.RsAnnotatorTestBase
+
+class AddSelfTest : RsAnnotatorTestBase() {
+    @Test
+    fun testNoParameters() = checkQuickFix("Add self to function", """
```

Comment:
No need to annotate tests with `@Test`. Also, we are switching to the more readable test names format, and I suggest we use it here:

```Rust
fun `test no parameters`() = check(..)
```